docs/manual/java/guide/production/code/docs/production/ConfigurationServiceLocatorModule.java

```diff
@@ -6,17 +6,24 @@
 import com.lightbend.lagom.javadsl.client.ConfigurationServiceLocator;
 import play.Configuration;
 import play.Environment;
+import com.typesafe.config.Config;
 
 public class ConfigurationServiceLocatorModule extends AbstractModule {
 
     private final Environment environment;
-    private final Configuration configuration;
 
-    public ConfigurationServiceLocatorModule(Environment environment, Configuration configuration) {
+    private final Config configuration;
+
+    public ConfigurationServiceLocatorModule(Environment environment, Config configuration) {
         this.environment = environment;
         this.configuration = configuration;
     }
 
+    @Deprecated
```

Comment:
This code is only used for the documentation snippet on https://www.lagomframework.com/documentation/1.3.x/java/ProductionOverview.html#Deploying-using-static-service-locations so the deprecated constructor can be removed here, too.

(Why we give people a snippet of code to copy and paste rather than providing this module as part of the framework, I have no idea. Maybe a future improvement possibility.)